USAGE-DETAILS.md

```diff
@@ -4,130 +4,131 @@
   resources:
     requests:
       storage: 1Gi
   storageClassName: gold
+      
 
-
-$> kubectl apply -f demo-pvc-gold.yaml
-persistentvolumeclaim/demo-pvc created
+$> kubectl apply -f demo-pvc-file-system.yaml
+persistentvolumeclaim/demo-pvc-file-system created
 ```
 
 View the PVC and the created PV:
 
 ```sh
 $> kubectl get pv,pvc
-NAME                                                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS   REASON   AGE
-persistentvolume/pvc-a04bd32f-bd0f-11e9-a1f5-005056a45d5f   1Gi        RWO            Delete           Bound    default/demo-pvc   gold                    78s
+NAME                                                    CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS   REASON   AGE
+persistentvolume/pvc-828ce909-6eb2-11ea-abc8-005056a49b44   1Gi        RWO            Delete           Bound    default/demo-pvc   gold                    78s
 
-NAME                             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-persistentvolumeclaim/demo-pvc   Bound    pvc-a04bd32f-bd0f-11e9-a1f5-005056a45d5f   1Gi        RWO            gold           78s
+NAME                                         STATUS   VOLUME                                 CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+persistentvolumeclaim/demo-pvc-file-system   Bound    pvc-828ce909-6eb2-11ea-abc8-005056a49b44   1Gi        RWO            gold           78s
 
 
-$> kubectl describe persistentvolume/pvc-a04bd32f-bd0f-11e9-a1f5-005056a45d5f
-Name:            pvc-a04bd32f-bd0f-11e9-a1f5-005056a45d5f
+$> kubectl describe persistentvolume/pvc-828ce909-6eb2-11ea-abc8-005056a49b44
+Name:            pvc-828ce909-6eb2-11ea-abc8-005056a49b44
 Labels:          <none>
 Annotations:     pv.kubernetes.io/provisioned-by: block.csi.ibm.com
-Finalizers:      [kubernetes.io/pv-protection]
+Finalizers:      [kubernetes.io/pv-protection external-attacher/block-csi-ibm-com]
 StorageClass:    gold
 Status:          Bound
-Claim:           default/demo-pvc
+Claim:           default/demo-pvc-file-system
 Reclaim Policy:  Delete
 Access Modes:    RWO
 VolumeMode:      Filesystem
 Capacity:        1Gi
 Node Affinity:   <none>
-Message:         
+Message:
 Source:
     Type:              CSI (a Container Storage Interface (CSI) volume source)
     Driver:            block.csi.ibm.com
-    VolumeHandle:      A9000:6001738CFC9035EB0000000000D1F68F
+    VolumeHandle:      SVC:60050760718106998000000000000543
     ReadOnly:          false
-    VolumeAttributes:      array_address=<IP>
-                           pool_name=gold
-                           storage.kubernetes.io/csiProvisionerIdentity=1565550204603-8081-block.csi.ibm.com
-                           storage_type=A9000
-                           volume_name=demo1_pvc-a04bd32f-bd0f-11e9-a1f5-005056a45d5f
+    VolumeAttributes:      array_address=baremetal10-cluster.xiv.ibm.com
```

Comment:
I don't think we should put this on the internet

Reply:
Changed to <IP> as it wasbefore